.github/workflows/build_and_publish_changed_images.yaml

```diff
@@ -1,10 +1,357 @@
+          images/app-streaming/nvidia_icd.json
+
+    - id: check_desktop
+      name: Check for changes to the desktop image source
+      uses: tj-actions/changed-files@v1.1.2
+      with:
+        files: images/desktop
+
+    - id: check_pulseaudio
+      name: Check for changes to the pulseaudio image source
+      uses: tj-actions/changed-files@v1.1.2
+      with:
+        files: images/pulseaudio
+
+    - id: check_squid_proxy
+      name: Check for changes to the squid proxy image source
+      uses: tj-actions/changed-files@v1.1.2
+      with:
+        files: images/squid-proxy
+
+    - id: check_tinyfilemanager
+      name: Check for changes to the tinyfilemanager image source
+      uses: tj-actions/changed-files@v1.1.2
+      with:
+        files: images/tinyfilemanager
+
+    - id: check_uinput_device_plugin
+      name: Check for changes to the uinput device plugin image source
+      uses: tj-actions/changed-files@v1.1.2
+      with:
+        files: images/uinput-device-plugin
+
+    - id: check_watchdog
+      name: Check for changes to the watchdog image source
+      uses: tj-actions/changed-files@v1.1.2
+      with:
+        files: images/watchdog
+
+    - id: check_webrtc
+      name: Check for changes to the webrtc image source
+      uses: tj-actions/changed-files@v1.1.2
+      with:
+        files: images/webrtc
+
+    - id: check_xpra
+      name: Check for changes to the xpra image source
+      uses: tj-actions/changed-files@v1.1.2
+      with:
+        files: images/xpra
+
+    - id: check_xserver
+      name: Check for changes to the xserver image source
+      uses: tj-actions/changed-files@v1.1.2
+      with:
+        files: images/xserver
+
+  app_proxy:
+    needs: check_image_source_changes
+    if: needs.check_image_source_changes.outputs.app_proxy_changed == 'true'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build & publish image
+      uses: ./.github/actions/build_and_publish_image
+      with:
+        github_token: $github_token
+        github_username: $github_username
+        image_name: app-proxy
+        image_source_directory: images/app-proxy
+        image_version_1: ${{ inputs.image_version_1 }}
+
+  app_streaming_bionic:
+    needs: check_image_source_changes
+    if: needs.check_image_source_changes.outputs.app_streaming_bionic_changed == 'true'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build & publish image
+      uses: ./.github/actions/build_and_publish_image
+      with:
+        github_token: $github_token
+        github_username: $github_username
+        image_name: app-streaming
+        image_source_directory: images/app-streaming
+        image_version_1: ${{ inputs.image_version_1 }}
+        image_version_suffix: -bionic
+
+  app_streaming_buster:
+    needs: check_image_source_changes
+    if: needs.check_image_source_changes.outputs.app_streaming_changed_buster == 'true'
```

Comment:
typo, change `app_streaming_changed_buster` to `app_streaming_buster_changed`

Reply:
Whoops, good catch

Reply:
Done